### PR TITLE
Adaptive layout of the block top

### PR DESCRIPTION
**Pull request by NataliPchelka**

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -147,79 +148,144 @@ body {
...
+  font-size: 18px;
+  font-weight: 400;
+}
+
+.top__lable-check::after {
+  display: block;
+  width: 15px;
+  content: "";
+  position: absolute;
+  border: 1px solid #FFFFFF;
```
Comment (almuly) [inline styles/styles.css]: use your varibles for this color

File: styles/styles.css [hunk 2]
```diff
@@ -147,79 +148,144 @@ body {
...
+}
+
+.top__number-people {
+  width: 30%;
+  border: none;
+}
+
+.top__number-people::placeholder {
+  font-size: 18px;
+  color: #383838;
```
Comment (almuly) [inline styles/styles.css]: use your varibles for this color

File: styles/styles.css [hunk 3]
```diff
@@ -435,6 +554,154 @@ body {
...
+    flex-direction: column;
+    margin-block-start: 0px;
+    background-color: inherit;
+    padding-bottom: 50px;
+  }
+
+  .top__input-name {
+    height: 48px;
+    border-radius: 0;
+    border: 4px solid var(--font-bright);
```
Comment (almuly) [inline styles/styles.css]: Your border size is right for top and bottom.  but for left and right it must be 8px. So it's touched all you left and right borders on mobile form

**Comment by almuly** [comment]
Godd work, just small fixes and it will be look great!

File: styles/styles.css (hunk 1 above)
Comment (NataliPchelka) [reply styles/styles.css]: Fixed

File: styles/styles.css (hunk 2 above)
Comment (NataliPchelka) [reply styles/styles.css]: Fixed

File: styles/styles.css (hunk 3 above)
Comment (NataliPchelka) [reply styles/styles.css]: Fixed

**Pull request merged**